src/main/java/org/truffleruby/cext/CExtNodes.java

```diff
@@ -877,9 +877,8 @@ protected Object callSuper(VirtualFrame frame, Object[] args) {
             final InternalMethod callingMethod = RubyArguments.getMethod(callingMethodFrame);
             final Object callingSelf = RubyArguments.getSelf(callingMethodFrame);
             final DynamicObject callingMetaclass = metaClassNode.executeMetaClass(callingSelf);
-            final DeclarationContext declarationContext = RubyArguments.getDeclarationContext(frame);
             final MethodLookupResult superMethodLookup = ModuleOperations
-                    .lookupSuperMethod(callingMethod, callingMetaclass, declarationContext);
+                    .lookupSuperMethod(callingMethod, callingMetaclass);
```

Comment:
Could you clarify why we don't need to pass the frame's DeclarationContext here anymore?

Reply:
Because it's always the same as `callingMethod.getDeclarationContext()`, I see.

Reply:
Yep :) when I added this new argument, I did not know about it